src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -208,6 +210,12 @@ void GcsResourceManager::HandleGetAllResourceUsage(
     const rpc::GetAllResourceUsageRequest &request,
     rpc::GetAllResourceUsageReply *reply,
     rpc::SendReplyCallback send_reply_callback) {
+  if (!local_node_id_.IsNil() && RayConfig::instance().gcs_actor_scheduling_enabled()) {
```

Comment:
When will this be NIL? Should we just CHECK it here? I feel the second condition should be enough?

Reply:
Yes, the Nil check could be removed.